### DISCUSSION TITLE
Translate default error messages into another language.

### DISCUSSION THREAD

**Discussion by franklinvillalobos** [leafsphp/leaf#223] [Q&A]
Hello, I want to know how I can translate leaf's error messages into another language.
Where is the file to add or change the language?

**Comment by fadrian06** [comment]
What I do for those cases, I develop applications in Spanish . Then I create a spanish.json or a spanish.php. Whatever as long as you export an array with the translations of every Leaf Form rule and then I pass that array to the Form::message() method
